core/src/main/kotlin/org/evomaster/core/problem/rest/OpenApiAccess.kt

```diff
@@ -17,6 +19,8 @@ import javax.ws.rs.core.Response
  */
 object OpenApiAccess {
 
+    val unauthorizedSchemaAccess : String = "UNAUTHORIZED_SCHEMA_ACCESS"
+
```

Comment:
constant variables like this should be in upper-snake-case, ie, `val UNAUTHORIZED_SCHEMA_ACCESS`

Reply:
Hi Andrea,

Thanks for your comment. I have fixed this one.